apps/tables/src/components/ViewTable.vue

```diff
@@ -25,6 +26,20 @@ export default {
     RoutedTableExplorer,
   },
   computed: {
+    canView() {
+      if (this.session) {
```

Comment:
anonymous users have no session?

Reply:
jawel, maar is wellicht nog niet geladen. Maar dat kan vast nog mooier.

Reply:
nog ff gekeken maar dit is denk ik zoals het moet. Het is reactive dus zou opnieuw moeten vuren als session verandert toch?